### ISSUE TITLE
Breaking change in unpinned pyperclip dependency

### ISSUE THREAD

**Issue by chair6** [python-cmd2/cmd2#230]
On trying to import a new cmd2 install on a Ubuntu system where pyperclip cannot access the clipboard (triggering the failure path in https://github.com/python-cmd2/cmd2/blob/master/cmd2.py#L326), I'm getting the following traceback:
```
  File "[snip]/__init__.py", line 9, in <module>
    from cmd2 import Cmd
  File "/home/ubuntu/virtualenvs/venv-2.7.10/lib/python2.7/site-packages/cmd2.py", line 333, in <module>
    except pyperclip.exceptions.PyperclipException:
AttributeError: 'module' object has no attribute 'exceptions'
```

It seems cmd2's unpinned pyperclip dependency is pulling in a breaking-change pyperclip release from yesterday.  In https://github.com/asweigart/pyperclip/commit/df268c11d1b9bdfcb322e7fdcb61191e572e316c, pyperclip was restructured into a single file meaning pyperclip.exceptions.PyperclipException no longer resolves.

**Comment by tleonhardt** [comment]
@chair6 Thanks for bringing this to our attention.  We will work on a fix for it shortly.

**Comment by tleonhardt** [comment]
@chair6 This issue has been fixed in ``cmd2`` version **0.7.8**, which has been released and is available now from PyPI

**Comment by chair6** [comment]
Thanks, @tleonhardt!! The quick turnaround is greatly appreciated...
